packages/acceptance-tests/pkg-tests-specs/sources/commands/publish.test.ts

```diff
@@ -37,8 +37,38 @@ describe(`npmPublishUtils.getGitHead`, () =>   {
 
       await run(`npm`, `publish`, {
         env: {
-          YARN_NPM_AUTH_TOKEN: `686159dc-64b3-413e-a244-2de2b8d1c36f`,
+          YARN_NPM_AUTH_TOKEN: `316158de-64b3-413e-a244-2de2b8d1c80f`,
         },
       });
     }));
 });
+
+describe(`--otp`, () =>   {
+  test(`should fail when invalid otp is given`,
```

Comment:
I've also wanted to add a test that would verify that you are prompting for the OTP but the testing helper doesn't allow you to hook into stdin/stdout stuff easily and, unfortunately, I didn't have time to add this.